File: Programmers/Lv0/주사위게임1/solution.py
def is_odd(num: int):
    return num % 2 == 1


def solution(a, b):
    if is_odd(a) and is_odd(b):
        return a ** 2 + b ** 2
    elif is_odd(a) or is_odd(b):
        return 2 * (a + b)

    return abs(a - b)

File: Programmers/Lv0/주사위게임1/Solution.java
package Lv0.주사위게임1;

class Solution {
    private boolean isOdd(int num) {
        return num % 2 == 1;
    }

    public int solution(int a, int b) {
        if (isOdd(a) && isOdd(b)) return a * a + b * b;
        else if (isOdd(a) || isOdd(b)) return 2 * (a + b);
        return Math.abs(a - b);
    }
}
